public/main.js

```diff
@@ -0,0 +1,17 @@
+var helpful = 0;
+var unhelpful = 0;
+var total = 0;
+var display = document.getElementById("displayCount");
+var totalNumber = document.getElementById("totalNumber")
+document.getElementById("Button1").addEventListener("click", function (event) {
+    //var temp = event.target;
+    helpful++;
+    total++;
+    display.innerHTML = helpful;
+    totalNumber.innerHTML = total;
+});
+document.getElementById("Button2").addEventListener("click", function (event) {
+    unhelpful++;
+    total++;
+    totalNumber.innerHTML = total;
+});
```

Comment:
You're going to run into problems using `getElementById`, because every `id` on an HTML page must be completely unique. You can't have two elements with the same `id`.

Learn about [querySelectorAll](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll) which allows you to select by class name and other attributes.

You'll face the same problem with your global `helpful` and `total` vars. If each question has it's own values, you'll need a way to store those values separately.